pkg/syncthing/syncthing.go

```diff
@@ -267,7 +262,39 @@ func (s *Syncthing) waitForPing(ctx context.Context, wg *sync.WaitGroup, local b
 	return fmt.Errorf("Syncthing not responding after 50s")
 }
 
-func (s *Syncthing) waitForScanning(ctx context.Context, wg *sync.WaitGroup, dev *model.Dev, local bool) error {
+//SendStignoreFile sends .stignore from local to remote
+func (s *Syncthing) SendStignoreFile(ctx context.Context, wg *sync.WaitGroup, dev *model.Dev) {
+	log.Infof("Sending '.stignore' file to the remote container...")
+	folder := fmt.Sprintf("okteto-%s", dev.Name)
+	params := map[string]string{"folder": folder}
+	ignores := &Ignores{}
+	body, err := s.APICall("rest/db/ignores", "GET", 200, params, true, nil)
+	if err != nil {
+		log.Infof("error getting 'rest/db/ignores' syncthing API: %s", err)
+		log.Errorf("'.stignore' not sent to the remote container: %s", err)
+		return
+	}
+	err = json.Unmarshal(body, ignores)
+	if err != nil {
+		log.Infof("error unmarshaling 'rest/db/ignores': %s", err)
+		log.Errorf("'.stignore' not sent to the remote container: %s", err)
+		return
+	}
+	body, err = json.Marshal(ignores)
+	if err != nil {
+		log.Infof("error marshaling 'rest/db/ignores': %s", err)
+		log.Errorf("'.stignore' not sent to the remote container: %s", err)
+	}
+	_, err = s.APICall("rest/db/ignores", "POST", 200, params, false, body)
```

Comment:
took me a bit to catch that we were using `true` vs `false`. could you add a comment mentioning that this is to the remote? (or maybe have two functions, `LocalAPICall` and `RemoteAPICall` so is clearer?)